submit_release_job.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# A simple script to submit the Argo workflow to build the release.
+#
+# Usage submit_release_job.sh ${COMMIT}
+#
+# COMMIT=commit to build at
+set -ex
+
+COMMIT=$1
+
+JOB_NAME="tf-operator-release"
+JOB_TYPE=tf-operator-release
+BUILD_NUMBER=$(uuidgen)
+BUILD_NUMBER=${BUILD_NUMBER:0:4}
+REPO_OWNER=kubeflow
+REPO_NAME=tf-operator
+ENV=releasing
+DATE=`date +%Y%m%d`
+PULL_BASE_SHA=${COMMIT:0:8}
+VERSION_TAG="v${DATE}-${PULL_BASE_SHA}"
+
+PROW_VAR="JOB_NAME=${JOB_NAME},JOB_TYPE=${JOB_TYPE},REPO_NAME=${REPO_NAME}"
+PROW_VAR="${PROW_VAR},REPO_OWNER=${REPO_OWNER},BUILD_NUMBER=${BUILD_NUMBER}" 
+PROW_VAR="${PROW_VAR},PULL_BASE_SHA=${PULL_BASE_SHA}" 
+
+cd test/workflows
```

Comment:
nit:  maybe `cd $(git rev-parse --show-toplevel)/test/workflows` so developers could run this from anywhere in the tree (or keep it in their PATH, even).